test/forms/PropertyLandValueTrustFormProviderSpec.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.behaviours.StringFieldBehaviours
+import forms.property_or_land.PropertyLandValueTrustFormProvider
+import play.api.data.FormError
+import wolfendale.scalacheck.regexp.RegexpGen
+
+class PropertyLandValueTrustFormProviderSpec extends StringFieldBehaviours {
+
+  val form = new PropertyLandValueTrustFormProvider()()
+
+  ".field1" must {
+
+    val fieldName = "value"
+    val requiredKey = "propertyLandValueTrust.error.field1.required"
+    val lengthKey = "propertyLandValueTrust.error.field1.length"
+    val maxLength = 12
+
```

Comment:
Need to add a behaviour for a non-decimal field?
(See IntFieldBehaviours.nonDecimalField)